lib/finite_machine/logger.rb

```diff
@@ -30,8 +30,8 @@ def format_error(error)
       message.join
     end
 
-    def report_transition(name, from, to, *args)
-      message = ["Transition: @event=#{name} "]
+    def report_transition(fsm_name, event_name, from, to, *args)
+      message = ["StateMachine: #{fsm_name} Transition: @event=#{event_name} "]
```

Comment:
I wonder if we could find a version that is a bit shorter and potentially easier to parse? For example, maybe we could have the machine name as an attribute of transition:

```
Transition @machine=... @event=...
```

Reply:
Ok. Let me try that out! Thanks.